Plugins/MeshFeatureTracking/src/MeshFeatureTrackingComponent.cpp

```diff
@@ -585,41 +585,38 @@ Scalar MeshFeatureTrackingComponent::getFeatureScale() const {
     auto ro = getRoMgr()->getRenderObject( m_pickedRoIdx );
     if ( m_pickedMesh->getRenderMode() == MeshRenderMode::RM_POINTS )
     { return ro->getAabb().sizes().norm() / 500; }
+    const auto& T = ro->getTransform();
```

Comment:
why not storing explicitly the ref to the linear part of the transformation. 
`const auto& T = ro->getTransform().linear()`.
Also beware that there is a lot of 'const auto& T`or 'const auto& t` (not upper and lower case for the `t`).
It could be a good idea to homogenize lowercase/uppercase notation (ref. coding style). Note also that `t`or `T` sometime refer to triangle, sometime to transform and it could be quite confusing to read the code with such semantic variation of a variable name.